frontend/src/calculations/FieldAndSoil/Crops/Calculations.ts

```diff
@@ -681,17 +548,24 @@ export async function sharedCalcCropReq(
     );
     nutrientValues = extractNutrientValues(nutrients);
   } else {
+    if (!phosphorousRegion || !potassiumRegion) {
```

Comment:
If you're curious, the berry bug fix was to move the error throwing here. We don't need these regions for berries.